cloudformation.yaml

```diff
@@ -63,14 +63,6 @@ Resources:
               Action:
                 - lambda:InvokeFunction
               Resource: "*"
-        - PolicyName: unEncryptedDynamoTable
-          PolicyDocument:
-            Statement:
-              Effect: Allow
-              Action:
-              - dynamodb:PutItem
-              - dynamodb:GetItem
-              Resource: !Sub 'arn:aws:dynamodb:${AWS::Region}:${AWS::AccountId}:table/cas-auth-${Stage}'
```

Comment:
we don't really need to keep the permissions to the unencrypted table since no one is using this lambda in prod yet so it doesn't matter if it stops working in the time between cloudformation update and new code deployment